### DISCUSSION TITLE
Ability to drag file into Terminal and prefill path on Linux

### DISCUSSION THREAD

**Discussion by acidprime** [ghostty-org/ghostty#3928] [Feature Requests, Ideas] [answered]
I am using Ubuntu 24.04 and I noticed that one of the main features missing is the ability to drag files and directories into the Terminal window and have them pre-fill the paths.

**Comment by rhodes-b** [comment]
this works with nautilus and dolphin you need the python package that allows nautilus addons im not sure if there are any steps to get it working in KDE

**Reply by 00-kat** [reply, marked as answer]
> KDE

xref #7742.
